.editorconfig

```diff
@@ -0,0 +1,34 @@
+# This is the top-most EditorConfig file.
+root = true
+
+# For all files.
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# Shell scripts
+[*.sh]
+indent_size = 4
+
+# Python files
+[*.py]
+indent_size = 4
+
+# .cirrus.yml, .appveyor.yml, .fuzzbuzz.yml, etc.
+[*.yml]
+indent_size = 2
+
+# Makefiles
+[{*.am,Makefile.*.include}]
+indent_style = tab
+
+# Autoconf scripts
+[configure.ac]
+indent_size = 2
+
+# C++ files
+[*.{h,cpp}]
```

Comment:
The `ident_size` is common to all the source code files defined in this editor config file. It could be combined:

```suggestion
# Source code files
[*.{h,cpp,py,sh}]
```

Reply:
Yes, thanks